root/static/scripts/common/coverart.js

```diff
@@ -25,7 +25,8 @@ $(function () {
         }).attr({
             "src": thumbnail_url,
             "title": $e.data("title")
-        }).appendTo(this);
+        })
+            .appendTo(this);
```

Comment:
This part looks weird now, since the `.bind` and `.attr` calls above don't have the same indentation - we should move those to separate lines and indent them like here.

There are a lot of other cases like this in the PR, though I won't leave comments on all of them.